Core/include/Acts/TrackFitting/BetheHeitlerApprox.hpp

```diff
@@ -0,0 +1,324 @@
+// This file is part of the Acts project.
+//
+// Copyright (C) 2021 CERN for the benefit of the Acts project
+//
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+#pragma once
+
+#include "Acts/Definitions/Algebra.hpp"
+#include "Acts/TrackFitting/detail/GsfUtils.hpp"
+
+#include <array>
+#include <fstream>
+#include <mutex>
+#include <random>
+
+#include <boost/container/static_vector.hpp>
+
+namespace Acts {
+
+namespace detail {
+
+struct GaussianComponent {
+  ActsScalar weight = 0.0;
+  ActsScalar mean = 0.0;
+  ActsScalar var = 0.0;
+};
+
+/// Transform a gaussian component to a space where all values are defined from
+/// [-inf, inf]
+void transformComponent(const GaussianComponent &cmp,
+                        double &transformed_weight, double &transformed_mean,
+                        double &transformed_var) {
+  const auto &[weight, mean, var] = cmp;
+
+  transformed_weight = std::log(weight) - std::log(1 - weight);
+  transformed_mean = std::log(mean) - std::log(1 - mean);
+  transformed_var = std::log(var);
+}
+
+/// Transform a gaussian component back from the [-inf, inf]-space to the usual
+/// space
+auto inverseTransformComponent(double transformed_weight,
+                               double transformed_mean,
+                               double transformed_var) {
+  GaussianComponent cmp;
+  cmp.weight = 1. / (1 + std::exp(-transformed_weight));
+  cmp.mean = 1. / (1 + std::exp(-transformed_mean));
+  cmp.var = std::exp(transformed_var);
+
+  return cmp;
+}
+
+}  // namespace detail
+
+namespace Experimental {
+
+/// This class approximates the Bethe-Heitler with only one component. This is
+/// mainly inside @ref AtlasBetheHeitlerApprox, but can also be used as the
+/// only component approximation (then probably for debugging)
+struct BetheHeitlerApproxSingleCmp {
+  /// Returns the number of components the returned mixture will have
+  constexpr auto numComponents() const { return 1; }
+
+  /// Checks if an input is valid for the parameterization. The threshold for
```

Comment:
```suggestion
  /// Checks if an input is valid for the parameterisation. The threshold for
```

Reply:
I would actually stick to the AE spelling here...